docs/backends/pktvisor.md

```diff
@@ -0,0 +1,99 @@
+# Pktvisor
+The `pktvisor` backend embeds the [pktvisord](https://github.com/netboxlabs/pktvisor) process inside Orb Agent to run deep packet analytics, active probes, and streaming aggregations directly at the edge. Pktvisor policies let you decide which network taps to activate, which analyzers to run, and how the resulting metrics are exported to your observability stack.
+
+## Configuration
+Orb writes a temporary pktvisor configuration file on startup based on the `orb.backends.pktvisor` block. Any key that is not handled explicitly is forwarded to `visor.config` in the generated file, so you can pass through native pktvisor options such as logging, crashpad, or custom data paths when needed.
+
+### Backend settings
+| Parameter | Type | Required | Default | Description |
+|:---------:|:----:|:--------:|:-------:|-------------|
+| `host` | string | no | `localhost` | Admin API host exposed by `pktvisord`. Use this when `pktvisord` runs outside the Orb container. |
+| `port` | string | no | `10853` | Admin API port. Must match the listener configured for `--admin-api`. |
+| `taps` | map | yes | – | Declarative tap definitions copied into `visor.taps`. Each tap sets the data source (`input_type`, `config`, `filter`, and optional `tags`). |
+| *other keys* | any | no | – | Added verbatim under `visor.config` (for example `log_level`, crashpad options, or storage paths). |
+
+Pktvisor ships with the Orb agent container image. If you run Orb on a bare host, ensure the `pktvisord` binary is in `$PATH` or adjust your deployment accordingly.
+
+### Exporting pktvisor metrics
+`pktvisord` can stream OpenTelemetry HTTP metrics directly to a collector. Configure the shared backend section to point Orb Agent at your collector endpoint:
+
+```yaml
+orb:
+  backends:
+    pktvisor:
+      host: 0.0.0.0
+      port: "10853"
+      taps:
+        dns_pcap:
+          input_type: pcap
+          config:
+            iface: eth0
+          filter:
+            bpf: "port 53"
+        sflow:
+          input_type: flow
+          config:
+            flow_type: sflow
+            port: 6343
+            bind: 192.168.1.1
+          tags:
+            virtual: false
+            vhost: 2
+    common:
+      otlp:
+        http: "http://otel-collector.monitoring.svc.cluster.local:4318"
```

Comment:
Should we line this up to match the opentelemetry_infinity docs? e.g. `http://0.0.0.0:4320`